src/mad_qc.py

```diff
@@ -53,8 +55,10 @@ def main(args):
     logger.info('Running madQC command %s', run_cmd)
     mad_output = subprocess.check_output(shlex.split(run_cmd))
     os.rename('MAplot.png', plot_output_filename)
-    qc_metrics = dict()
-    qc_metrics['MAD.R'] = json.loads(mad_output.decode())
+    qc_metrics = {}
```

Comment:
Are you satisfied existing unit test covers this?

Reply:
You make a good point. I will actually import the QC metric machinery from the rna_qc and use that. I'll also add the madqc.json into md5testing (i seem to have stopped this change halfway, the whole point of ordering this dict was to get md5 equivalent results every time..).